node/cn/handler.go

```diff
@@ -461,7 +461,7 @@ func (pm *ProtocolManager) processConsensusMsg(msgCh <-chan p2p.Msg, p Peer, add
 			_, err := handler.HandleMsg(addr, msg)
 			// if msg is istanbul msg, handled is true and err is nil if handle msg is successful.
 			if err != nil {
-				p.GetP2PPeer().Log().Error("ProtocolManager failed to handle consensus message", "msg", msg, "err", err)
+				p.GetP2PPeer().Log().Warn("ProtocolManager failed to handle consensus message. This can happen during block synchronization.", "msg", msg, "err", err)
```

Comment:
I'm not sure we can check the block sync here but how about logging error if block sync is not happened?

Reply:
@jeonghonah At the moment, the only error that can be returned by `HandleMsg` is `ErrStoppedEngine` and the error only can be happened when a downloader stopped the consensus engine.

I agree that this can be improved if other error cases are added, but I don't think it can be done at the moment